gwsurrogate/surrogate.py

```diff
@@ -155,7 +155,7 @@ def __call__(self, q, M=None, dist=None, phi_ref=None,\
        times           --- array of times at which surrogate is to be evaluated
        units           --- units (mks or dimensionless) of input array of time samples
        singlemode_call --- Never set this by hand, will be False if this routine is called by the multimode evaluator
-
+       calibrated      --- If set to false, the calibration coefficients alpha and beta are set to 1
```

Comment:
`calibration` is only applicable for the models EMRISur1dq1e4 and BHPTNRSur1dq1e4. Can you please add a short remark about this in the docstring?